InteractiveHtmlBom/web/ibom.css

```diff
@@ -227,6 +227,14 @@ canvas:active {
   background-color: #313b40;
 }
 
+.bom tr.checked {
+  color: #AAA;
```

Comment:
nit: use small case for style consistency

Reply:
Done.